File: qe-selenium-python/activity_nine.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.common.action_chains import ActionChains

driver = webdriver.Firefox()
driver.get("https://training-support.net/webelements/keyboard-events")
driver.maximize_window()

print(driver.title)

actions = ActionChains(driver)
actions.send_keys("string from Selenium script").perform()


result = driver.find_element(
    By.XPATH, "//p[@id='result']/following-sibling::h1"
)
print(result.text)

driver.quit()

File: qe-selenium-java/src/main/java/demos/ActivityNine.java
package demos;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.interactions.Actions;

public class ActivityNine {
    public static void main(String[] args) throws InterruptedException {

        WebDriver driver = new FirefoxDriver();
        driver.get("https://training-support.net/webelements/keyboard-events");
        driver.manage().window().maximize();

        System.out.println(driver.getTitle());

        Actions actions = new Actions(driver);

        actions.sendKeys("string from Selenium script").perform();

        WebElement result = driver.findElement(By.xpath("//p[@id='result']/following-sibling::h1"));
        System.out.println(result.getText());
        driver.quit();
    }
}
